foster55f/market2table-fe
# Vendor page

What does this PR do?
Creates VendorsPage and getVendorsByMarketId

creates VendorsPage for each individual Vendor with name, description , and products with prices
displays individual vendor pages

getVendorsByMarketId will get all individual vendorcards displayed for a market
each VendorCard can be clicked on and taken to VendorPage with page '/markets/:marketId/vendors/:vendorId'

zipCodeForm issue fixed for enter button to get all markets

How to test?
Pull down branch
run 'npm start' and the site should open in browser via localhost:3000

Issues:
No issues

Relevant Screenshots:
![2020-02-24 14 10 15](https://user-images.githubusercontent.com/50148342/75191469-9ee2a180-570f-11ea-9f7c-30a9e0441ec4.gif)
